src/pages/authenticationPage/Signup.js

```diff
@@ -0,0 +1,165 @@
+    }
+  };
+
+  return (
+    <>
+      <div className="login-page flex flex-center">
+        <form className="form" onSubmit={signupHandler}>
+          <div className="login-heading">SIGN UP</div>
+          <label htmlFor="name" className="form-label">
+            Name
+          </label>
+          <input
+            className="input"
+            type="text"
+            placeholder="Enter Name"
+            name="name"
+            required
+            value={userCredentials.firstName}
+            onChange={(event) =>
+              setUserCredentials({
+                ...userCredentials,
+                firstName: event.target.value,
+              })
+            }
+          />
+          <label htmlFor="email" className="form-label">
+            Email
+          </label>
+          <input
+            className="input"
+            type="text"
+            placeholder="Enter Email"
+            name="email"
+            required
+            value={userCredentials.email}
+            onChange={(event) =>
+              setUserCredentials({
+                ...userCredentials,
+                email: event.target.value,
+              })
+            }
+          />
+          <label htmlFor="password" className="form-label">
+            Password
+          </label>
+          <div className="show-hide-password">
+            <input
+              id="create-password"
+              className="input"
+              type="password"
+              placeholder="Enter Password"
+              name="password"
+              required
+              value={userCredentials.password}
+              onChange={(event) =>
+                setUserCredentials({
+                  ...userCredentials,
+                  password: event.target.value,
+                })
+              }
+            />
+            <div className="show-password" id="show-create-password">
+              show
+            </div>
+          </div>
+
+          <label htmlFor="confirmPassword" className="form-label">
+            Confirm Password
+          </label>
+          <div className="show-hide-password">
+            <input
+              id="confirm-password"
+              className="input"
+              type="password"
+              placeholder="Enter Password"
+              name="password"
+              required
+              value={userCredentials.confirmPassword}
+              onChange={(event) =>
+                setUserCredentials({
+                  ...userCredentials,
+                  confirmPassword: event.target.value,
+                })
+              }
+            />
+            <div className="show-password" id="show-confirm-password">
+              show
+            </div>
+          </div>
+          <div>{error}</div>
+          <div className="login-store flex flex-space-between">
+            <label htmlFor="store">
+              <input type="checkbox" />I accept all terms & conditions
```

Comment:
add required attribute to this checkbox also

Reply:
Done the changes